style/checkstyle.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <property name="charset" value="UTF-8"/>
+    <property name="severity" value="info"/>
```

Comment:
we can keep error level since we have fixed style issues in commons repo.
`<property name="severity" value="error"/>`

Reply:
I updated these to keep setting up to date with hugegraph.
we can discuss whether to use "error level" setting.
I think we can set error level.